lib/matplotlib/mathtext.py

```diff
@@ -771,6 +774,13 @@ def __init__(self, *args, **kwargs):
         prop = FontProperties('cmex10')
         font = findfont(prop)
         self.fontmap['ex'] = font
+        # include stix sized alternatives for glyphs if fallback is stix
+        if isinstance(self.cm_fallback, StixFonts):
+            stixsizedaltfonts = "STIXGeneral STIXSizeOneSym STIXSizeTwoSym "\
+                "STIXSizeThreeSym STIXSizeFourSym STIXSizeFiveSym".split()
+            for i, stixfont in enumerate(stixsizedaltfonts):
+                font = findfont(stixfont)
+                self.fontmap[i] = font
```

Comment:
I went for splitting a string here because that is also done in line 770.

```python
for texfont in "cal rm tt it bf sf".split():
# ...
```

But of course, there is no line break and the order does not matter. So, I agree that a dict is more appropriate here. I also made sure to add the STIX alternatives with their names to the fontmap

```python
self.fontmap[name] = fullpath
```

Not sure whether this is really necessary, but it is also the case for the `DejaVuFonts`.